### PR TITLE
[FIX] project: fix the project activities filter kanban view

### PR DESCRIPTION
**Pull request by Ephoniz**
Steps to reproduce:

- Go to Projects app.
- Add some activities to different project (for today, overdue, future).
- Go to the activities button in the navbar.
- Search for the project activities and click in 'Today'.
- We are redirected to the kanban view of the projects without filtering

Issue:

We don't get the project filters by it's activities as it should be.

Solution:

Added the proper filters to this specific project kanban view so we can access the filters when accessing the view.

FW - port: 16.0

opw-3095944

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
looks good to me

**Comment by nle-odoo** [comment]
side note: what does "`FW - port: 16.0`" mean? if you want it to not be forward-ported in a version, you need to send a message with "fw-bot up to [last version it needs to be forward-ported in]" or `fw-bot ignore` for no forward-port at all.

**Comment by Ephoniz** [comment]
It's a way to specify until which version should it be forwarded to, but I think you're right I might use another syntax or another way to express it, because for this issue for example, it affects all versions except master.

**Comment by nle-odoo** [comment]
no problem, then I mean you can just do:

fw-bot up to 16.0

(but maybe you were going to do it before the merge happen)

side note: checking, the seems to be a backport from master, the master PR was set to be merged in 15.0 but to prevent problem with customization, it was retargeted to master (link: https://github.com/odoo/odoo/pull/103526#issuecomment-1290254744)

But as for the comment in that PR:

> The problem is that if someone has made custo with same filter names, what would happen?

It seems that the system will do an OR between the multiple filters, so this seems to be working correctly to me:

> 2022-12-15-162552_706x574_scrot

so I'm not sure if this could bring any issue.

**Review by xavierbol — COMMENTED** [review]
LGTM, can you remove the `fw-port 16` in your commit message? 🙂 

Once it is done I will r+ your fix. 🙂 

Also, once you want a review from the RD Team you can mark your PR as ready for review to activate the code owners. 🙂

**Comment by Ephoniz** [comment]
> LGTM, can you remove the `fw-port 16` in your commit message? slightly_smiling_face
> 
> Once it is done I will r+ your fix. slightly_smiling_face
> 
> Also, once you want a review from the RD Team you can mark your PR as ready for review to activate the code owners. slightly_smiling_face

Done! Thanks for the review 😄

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work. 🙂 

robodoo r+

**Pull request closed without merging**
